client/settings/general-settings-section/remove-method-confirmation-modal.js

```diff
@@ -0,0 +1,59 @@
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import interpolateComponents from 'interpolate-components';
+import { Button } from '@wordpress/components';
+import PaymentMethodsMap from '../../payment-methods-map';
+import ConfirmationModal from 'wcstripe/components/confirmation-modal';
+import AlertTitle from 'wcstripe/components/confirmation-modal/alert-title';
+
+const RemoveMethodConfirmationModal = ( { method, onClose, handleRemove } ) => {
+	const { label } = PaymentMethodsMap[ method ];
+	return (
+		<ConfirmationModal
+			title={
+				<AlertTitle
+					title={ interpolateComponents( {
+						mixedString: __(
+							'Remove {{methodName /}} from checkout',
+							'woocommerce-gateway-stripe'
+						),
+						components: {
+							methodName: <span>&nbsp;{ label }&nbsp;</span>,
```

Comment:
If all we need is a string with a substituted variable, then we may not need an `interpolateComponents`. We can use [sprintf](https://github.com/WordPress/gutenberg/tree/trunk/packages/i18n#sprintf) from `'@wordpress/i18n'`
```
title={ sprintf(
						__(
							'Remove %s from checkout',
							'woocommerce-gateway-stripe'
						),
						label
					) }
```

One benefit is we don't need an extra `<span>` nor `&nbsp;`.

Reply:
Thanks for the suggestion :100: